vispy/app/application.py

```diff
@@ -77,6 +77,19 @@ def process_events(self):
         """
         return self._backend._vispy_process_events()
 
+    def sleep(self, duration_sec):
+        """
+        Sleep for the given duration. This is used to reduce
+        CPU stress when VisPy is run in interactive mode.
+        see inputhook.py for details
```

Comment:
I also use flake8, and so does Travis, so it's a good bet. The numpy
docstring stuff is separate, and I don't know of automated tools for that
other than our limited checking in test_docstring_parameters file. Should I
go ahead and test this on my systems?